src/main/java/games/strategy/engine/lobby/client/ui/LobbyGamePanel.java

```diff
@@ -95,15 +97,15 @@ private void createComponents() {
         .setPreferredWidth(130);
     m_gameTable.getColumnModel().getColumn(m_gameTableModel.getColumnIndex(LobbyGameTableModel.Column.Host))
         .setPreferredWidth(67);
-    m_gameTable.setDefaultRenderer(Date.class, new DefaultTableCellRenderer() {
+    m_gameTable.setDefaultRenderer(Instant.class, new DefaultTableCellRenderer() {
       private static final long serialVersionUID = -2807387751127250972L;
-      private final SimpleDateFormat format = new SimpleDateFormat("hh:mm a");
 
       @Override
       public Component getTableCellRendererComponent(final JTable table, final Object value, final boolean isSelected,
           final boolean hasFocus, final int row, final int column) {
         super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
-        setText(format.format((Date) value));
+        setText(TimeManager
+            .getLocalizedTimeWithoutSeconds(LocalDateTime.ofInstant((Instant) value, ZoneOffset.systemDefault())));
```

Comment:
:+1: Similar change as for `ChatMessagePanel#addChatMessage()`.  It looks like the text never escapes the UI, so we should be ok.